src/main/resources/webapp/js/controllers.js

```diff
@@ -1201,6 +1224,24 @@
             }, 10);
         };
 
+        $scope.getNextVisit = function (visit, rows) {
```

Comment:
I'm not sure if we should print different visit if the selected has actual date, maybe we could just disable the print button instead.
We need to ask Seth what he thinks about this.

Reply:
I will ask him about this.